src/fe/implicit/_Obj.js

```diff
@@ -1,46 +1,140 @@
+import * as _ from './_';
+import * as _Arr from './_Arr';
+/**
+ * Get keys of an object in an array.
+ * @param {Object} o
+ *
+ * @returns {Array} Array of keys
+ */
 export const keys = o => Object.keys(o || {});
+
+/**
+ * Create a new object using an object and the given props.
+ * @param {Object} obj
+ * @param {Object} props
+ *
+ * @returns {Object} New object
+ */
 export const create = (obj, props) => Object.create(obj, props);
+
+/**
+ * Check if the object has any key value pairs or is empty
+ * @param {Object} o
+ *
+ * @returns {boolean} Returns true or false if the object was empty or not
```

Comment:
The description for this feels pretty redundant.

Reply:
seems to me too. ok repairing it.